ldevnet/ldevnet.go

```diff
@@ -59,9 +60,12 @@ func (ld *LocalDevnet) Close() {
 	ld.closer()
 }
 
-func New(t *testing.T, numMiners int) (*LocalDevnet, error) {
+func New(numMiners int, blockDur time.Duration) (*LocalDevnet, error) {
```

Comment:
Add some extra changes to allow tuning the speed at which the local Testnet generates new blocks.
Also avoid being biased in running on _tests_, since we now support the _embeded mode_ which is running a local devote in a non-testing environment.